### PR TITLE
Bootstrap 5.3.0

### PR DESCRIPTION
**Pull request by neelavarshad**
Changed bootstrap version from 4.4.1 to 5.3.0

Review deployment and check all flows

### CODE REVIEW

**Review by santhoshb-msft — CHANGES_REQUESTED** [review]
Hi @neelavarshad could you merge latest from main and resolve conflicts

**Comment by neelavarshad** [comment]
@santhoshb-msft Merged and resolved

**Comment by santhoshb-msft** [comment]
> @santhoshb-msft Merged and resolved

Thank you @neelavarshad

**Comment by santhoshb-msft** [comment]
@neelavarshad some initial observations below

Admin page issues:

- Logo
- Menu notworking when minimized
- buttons alignment to right (details/activate page)
- Plans page tabs not working
- Email templates action not working
- Appsettings button gap
- Cookies title
- Fetch all moved
- Table alignment
- Offer page - space on the top.
- App config action not working.


Landing page: Yet to test

**Comment by neelavarshad** [comment]
- [ ] - Logo
- [ ] - Menu notworking when minimized
- [ ] - buttons alignment to right (details/activate page)
- [ ] - Plans page tabs not working
- [ ] - Email templates action not working
- [ ] - Appsettings button gap
- [ ] - Cookies title
- [ ] - Fetch all moved
- [ ] - Table alignment
- [ ] - Offer page - space on the top.
- [ ] - App config action not working.

File: src/CustomerSite/wwwroot/js/site.js
Comment (santhoshb-msft) [inline src/CustomerSite/wwwroot/js/site.js:1]: no changes here right?

**Review by santhoshb-msft — CHANGES_REQUESTED** [review]
@neelavarshad almost looks good with some imperfections in the UI which I will share directly with you. things like alignment and some button functionalities.

**Comment by santhoshb-msft** [comment]
@neelavarshad  once you are done with the changes, please assign the PR to me

**Comment by neelavarshad** [comment]
@santhoshb-msft Assigned the PR back to you. 
I have changed the UI per your comments. There was a missing screenshot for Change quantity, but I think you meant to showcase left alignment. I have made that change but let me know if there's anything else. Thanks!

File: src/CustomerSite/CustomerSite.csproj [hunk 1]
```diff
@@ -11,9 +11,17 @@
 
   <ItemGroup>
     <Compile Remove="Context\**" />
+    <Compile Remove="wwwroot\lib\bootstrap\4.4.1\**" />
```
Comment (santhoshb-msft) [inline src/CustomerSite/CustomerSite.csproj:14]: should we remove this?

**Review by santhoshb-msft — COMMENTED** [review]
a couple of code comments below and some ui changes suggested on the internal chat

File: src/CustomerSite/CustomerSite.csproj (hunk 1 above)
Comment (neelavarshad) [reply src/CustomerSite/CustomerSite.csproj:14]: This is just a compile/build file. I don't think we should overwrite anything here. Right?

**Comment by neelavarshad** [comment]
@santhoshb-msft  Pushed additional changes and assigned back to you. Thanks!

File: src/AdminSite/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -9,30 +9,30 @@
 
     <script src="~/lib/jquery/dist/jquery.js"></script>
     <script src="~/lib/jquery/dist/jquery.min.js"></script>
-    <link href="~/lib/bootstrap/4.4.1/css/bootstrap.min.css" rel="stylesheet" />
+    <link href="~/lib/Datatable/css/dataTables.bootstrap.min.css" rel="stylesheet" />
+    <link href="~/lib/Datatable/css/responsive.bootstrap.min.css" rel="stylesheet" />
```
Comment (santhoshb-msft) [inline src/AdminSite/Views/Shared/_Layout.cshtml:13]: Can you verify why we need this. I dont find them in the lib folder

**Review by santhoshb-msft — COMMENTED** [review]
comment inline

File: src/AdminSite/Views/Shared/_Layout.cshtml (hunk 2 above)
Comment (neelavarshad) [reply src/AdminSite/Views/Shared/_Layout.cshtml]: @santhoshb-msft  The 4.4.1 references are after the delete. Once merged, you won't see it. I've also unreferenced unused  Datatable stylesheet references. Thanks!

**Comment by neelavarshad** [comment]
Changes to landing page with Change quantity button added.

**Review by santhoshb-msft — APPROVED** [review]
tested with new installation and upgrade scenarios. 
All commented fixed by @neelavarshad 

Approving

**Pull request merged**